twingate/internal/provider/resource/group.go

```diff
@@ -3,14 +3,18 @@ package resource
 import (
 	"context"
 	"errors"
+	"fmt"
 	"log"
 
+	"github.com/Twingate/terraform-provider-twingate/twingate/internal/attr"
 	"github.com/Twingate/terraform-provider-twingate/twingate/internal/client"
 	"github.com/Twingate/terraform-provider-twingate/twingate/internal/model"
 	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
 	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
 )
 
+var ErrAllowedToChangeOnlyManualGroups = fmt.Errorf("allowed to change only %s groups", model.GroupTypeManual)
```

Comment:
@alexmensch please check this message

Reply:
Oh, I see. Okay! Let's do this:

"Only groups of type %s may be modified"

The problem is that I think most people don't know what a "MANUAL" group is for this error. Could we also show what the type of the group is where the attempt was made?

So something like this: ("Only groups of type %s may be modified. Group %s is a %s type group.", model.GroupTypeManual, <groupName>, <groupType>)

Reply:
sure

Reply:
updated error message